Could not invoke IntercomWrapper.updateUser

Hi, 
I'm with a problem with Android and RN 41. When I call the Intercom.updateUser I get this error:  ``Could not invoke IntercomWrapper.updateUser`` anyone know what could be wrong?? The function registerIdentifiedUser is working

Still some times I get this error: 
```
java.lang.RuntimeException: Could not invoke IntercomWrapper.updateUser
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.facebook.react.bridge.BaseJavaModule$JavaMethod.invoke(BaseJavaModule.java:333)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.facebook.react.cxxbridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:196)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:818)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.facebook.react.bridge.BaseJavaModule$JavaMethod.invoke(BaseJavaModule.java:319)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	... 8 more
08-28 10:41:30.418  7447  7520 E AndroidRuntime: Caused by: java.lang.NoSuchMethodError: No virtual method updateUser(Ljava/util/Map;)V in class Lio/intercom/android/sdk/Intercom; or its super classes (declaration of 'io.intercom.android.sdk.Intercom' appears in /data/app/dsappp/base.apk)
08-28 10:41:30.418  7447  7520 E AndroidRuntime: 	at com.robinpowered.react.Intercom.IntercomModule.updateUser(IntercomModule.java:92)
```

--- a/android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java
+++ b/android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java
@@ -19,6 +19,7 @@ import java.util.Map;
 import javax.annotation.Nullable;
 
 import io.intercom.android.sdk.Intercom;
+import io.intercom.android.sdk.UserAttributes;
 import io.intercom.android.sdk.identity.Registration;
 import io.intercom.android.sdk.push.IntercomPushClient;
 
@@ -220,11 +221,11 @@ public class IntercomModule extends ReactContextBaseJavaModule {
 
     private UserAttributes convertToUserAttributes(ReadableMap readableMap) {
         Map<String, Object> map = recursivelyDeconstructReadableMap(readableMap);
-        UserAttributes userAttributes = new UserAttributes.Builder();
+        UserAttributes.Builder builder = new UserAttributes.Builder();
         for (Map.Entry<String, Object> entry : map.entrySet()) {
-            userAttributes.withCustomAttribute(entry.getKey(), entry.getValue());
+            builder.withCustomAttribute(entry.getKey(), entry.getValue());
         }
-        return userAttributes.build();
+        return builder.build();
     }
 
 

--- a/android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java
+++ b/android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java
@@ -220,7 +220,7 @@ public class IntercomModule extends ReactContextBaseJavaModule {
 
     private UserAttributes convertToUserAttributes(ReadableMap readableMap) {
         Map<String, Object> map = recursivelyDeconstructReadableMap(readableMap);
-        UserAttributes userAttributes = new UserAttributes.Builder()
+        UserAttributes userAttributes = new UserAttributes.Builder();
         for (Map.Entry<String, Object> entry : map.entrySet()) {
             userAttributes.withCustomAttribute(entry.getKey(), entry.getValue());
         }

--- a/android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java
+++ b/android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java
@@ -88,8 +88,8 @@ public class IntercomModule extends ReactContextBaseJavaModule {
     @ReactMethod
     public void updateUser(ReadableMap options, Callback callback) {
         try {
-            Map<String, Object> map = recursivelyDeconstructReadableMap(options);
-            Intercom.client().updateUser(map);
+            UserAttributes userAttributes = convertToUserAttributes(options);
+            Intercom.client().updateUser(userAttributes);
             Log.i(TAG, "updateUser");
             callback.invoke(null, null);
         } catch (Exception e) {
@@ -216,6 +216,15 @@ public class IntercomModule extends ReactContextBaseJavaModule {
          Intercom.client().setBottomPadding(padding);
          Log.i(TAG, "setBottomPadding");
          callback.invoke(null, null);
+    }
+
+    private UserAttributes convertToUserAttributes(ReadableMap readableMap) {
+        Map<String, Object> map = recursivelyDeconstructReadableMap(readableMap);
+        UserAttributes userAttributes = new UserAttributes.Builder()
+        for (Map.Entry<String, Object> entry : map.entrySet()) {
+            userAttributes.withCustomAttribute(entry.getKey(), entry.getValue());
+        }
+        return userAttributes.build();
     }
 
 

